images/virtualization-artifact/pkg/controller/service/disk_service.go

```diff
@@ -296,7 +319,42 @@ func getAccessModeMax(modes []corev1.PersistentVolumeAccessMode) corev1.Persiste
 	return m
 }
 
-func (s DiskService) parseVolumeMode(status cdiv1.StorageProfileStatus) StorageCapabilities {
+func (s DiskService) parseVolumeMode(sc *storev1.StorageClass) (corev1.PersistentVolumeMode, bool) {
+	if sc == nil {
+		return "", false
+	}
+	switch sc.GetAnnotations()[common.AnnVirtualDiskVolumeMode] {
```

Comment:
There may not be annotations at all

Reply:
This works correctly. If the annotation does not exist, the default case will be processed.